src/quick-lint-js/fe/parse-statement.cpp

```diff
@@ -273,7 +273,12 @@ bool parser::parse_and_visit_statement(
       this->skip();
       this->check_body_after_label();
       goto parse_statement;
-
+      // "async export f()" is not valid. It should be "export async f()"
```

Comment:
Nit: Shouldn't this comment say 'async export **function** f' and 'export async **function** f'?